.github/workflows/tests.yaml

```
@@ -252,13 +171,19 @@ jobs:
             echo "DOCKER_GROUP=$DOCKER_GROUP" >> $GITHUB_ENV
       - name: set environment
         uses: HatsuneMiku3939/direnv-action@v1
+        with:
+          masks: ''
       - name: configure variables
         run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:3:40: Double quote to prevent globbing and word splitting

(shellcheck)